server.py

```
@@ -161,7 +252,38 @@ def _reload_langgraph_agent() -> tuple[bool, str]:
-            new_graph = create_agent_graph(new_config, knowledge_store=new_store)
+            # Reuse the running scheduler so a drawer save doesn't tear
+            # down the polling loop and orphan in-flight fires. Build
+            # one only when none was constructed yet — the typical
+            # path is: server boots before setup is complete (no
+            # scheduler), wizard finishes, drawer triggers reload —
+            # this is when we *first* construct the scheduler.
+            #
+            # Note: the freshly-built scheduler isn't started here.
+            # FastAPI's startup hook fires once at process start; on
+            # post-setup reloads we kick the polling loop manually.
+            global _scheduler
```

Ruff (0.15.12)

[warning] Using the global statement to update `_scheduler` is discouraged

(PLW0603)